app/Http/Controllers/Front/PostController.php

```diff
@@ -53,15 +49,8 @@ public function index()
     {
         $posts = $this->postRepository->getActiveOrderByDate($this->nbrPages);
 
-        $quote = Cache::remember('quote', 720, function(){
+        $quote = $this->quote_service->getQuote();
```

Comment:
Use it to get a quote, as that's our intent at this level, all other details are irrelevant.